src/backends/pytorch/libtorch_backend.cc

```diff
@@ -429,10 +434,15 @@ LibTorchBackend::Context::ReadFixedSizedOutputTensor(
         if (device_ == torch::kCPU) {
           memcpy(buffer, (char*)content + content_offset, expected_byte_size);
         } else {
+#ifdef TRTIS_ENABLE_GPU
           cudaMemcpy(
               buffer, (char*)content + content_offset, expected_byte_size,
               cudaMemcpyDeviceToHost);
         }
+#else
+          return Status(
+              RequestStatusCode::INTERNAL, "GPU instances not supported");
+#endif  // TRTIS_ENABLE_GPU
```

Comment:
move this block up, seems to be in wrong scope

Reply:
That's right. rectified.